Src/WitsmlExplorer.Frontend/components/Modals/ShowLogDataOnServerModal.tsx

```diff
@@ -0,0 +1,240 @@
+            onOptionsChange={onChangeServer}
+          />
+          <StyledTypography>Index range:</StyledTypography>
+          <label style={alignLayout}>
+            <Radio
+              id={"indexRangeFullRadio"}
+              name="indexRangeFullRadio"
+              value={IndexRangeOptions.Full}
+              onChange={handleIndexRangeOptionOnChange}
+              checked={indexRangeOption === IndexRangeOptions.Full}
+            />
+            <Typography>Full</Typography>
+          </label>
+          <label style={alignLayout}>
+            <Radio
+              id={"indexRangeSelectedRadio"}
+              name="indexRangeSelectedRadio"
+              value={IndexRangeOptions.Selected}
+              onChange={handleIndexRangeOptionOnChange}
+              checked={indexRangeOption === IndexRangeOptions.Selected}
+            />
+            <Typography>Selected</Typography>
+          </label>
+          <StyledTypography>Mnemonics:</StyledTypography>
+          <label style={alignLayout}>
+            <Radio
+              id={"mnemonicsAllRadio"}
+              name="mnemonicsAllRadio"
+              value={MnemonicsOptions.All}
+              onChange={handleMnemonicsOptionOnChange}
+              checked={mnemonicsOption === MnemonicsOptions.All}
+            />
+            <Typography>All</Typography>
+          </label>
+          <label style={alignLayout}>
+            <Radio
+              id={"mnemonicsSelectedRadio"}
+              name="mnemonicsSelectedRadio"
+              value={MnemonicsOptions.Selected}
+              onChange={handleMnemonicsOptionOnChange}
+              checked={mnemonicsOption === MnemonicsOptions.Selected}
+              disabled={isUrlTooLong}
+            />
+            <Typography>Selected</Typography>
+          </label>
+          {isUrlTooLong && (
+            <StyledBanner colors={colors}>
+              <Banner.Icon variant="warning">
+                <Icon name="infoCircle" />
+              </Banner.Icon>
+              <Banner.Message>
+                The selected number of mnemonics is too large to be saved in the
+                URL because the URL exceeds the maximum length of 2000
+                characters. As a result, it will not be possible to open this
+                URL on a different server to access the chosen mnemonics for the
+                given log. However, you can still open all mnemonics.
+              </Banner.Message>
+            </StyledBanner>
+          )}
+        </EdsProvider>
+      }
+      isLoading={isFetching}
+      onSubmit={() => handleOnSubmit()}
+      confirmText={"Show"}
+      switchButtonPlaces={false}
+      confirmDisabled={!selectedServer}
+    />
+  );
+}
+
+const alignLayout: CSSProperties = {
+  display: "flex",
+  alignItems: "center"
+};
+
+const StyledTypography = styled(Typography)<{ colors: Colors }>`
+  padding-top: 16px;
+`;
+
+const StyledBanner = styled(Banner)<{ colors: Colors }>`
```

Comment:
The StyledBanner is the same as a few other places. Would be nice to have it under StyledComponents, but it's fine if you leave it for later.